.changeset/tough-dodos-breathe.md

```diff
@@ -0,0 +1,5 @@
+---
+"pie-docs": minor
+---
+
+[Changed] sidebar styles
```

Comment:
Would be good to make this description a bit more descriptive – wouldn't mean much to me as to how it's been changed if I was just looking at the changelog.

Reply:
updated 💯